mkdocs.yml

```diff
@@ -36,7 +36,7 @@ nav:
     - Getting Started:
       - MetaMask: develop/start/metamask.md
       - Truffle: develop/start/truffle.md
-      - Hardhat: develop/start/hardhat.md
+      - HardHat: develop/start/hardhat.md
```

Comment:
I thought we agreed "Hardhat" was the correct spelling

Reply:
E.g. from [their website](https://hardhat.org/) "Hardhat is next-generation Ethereum tooling"

Reply:
Ops ... that commit was from an old branch. will fix it.